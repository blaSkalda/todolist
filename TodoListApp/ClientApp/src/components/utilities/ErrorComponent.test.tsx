import * as React from 'react';
import { shallow } from 'enzyme'

import Error from './ErrorComponent';

describe('<Error />', () => {

    it('renders <Error />', () => {
        const wrapper = shallow(<Error message='error' />);
        expect(wrapper).toMatchSnapshot();
    });
});
