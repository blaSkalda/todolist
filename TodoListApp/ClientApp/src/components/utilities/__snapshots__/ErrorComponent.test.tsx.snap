// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Error /> renders <Error /> 1`] = `
<Fragment>
  <div
    className="alert alert-danger"
    role="alert"
  >
    error
  </div>
</Fragment>
`;
